### PR TITLE
Feature/top 5 display

### PR DESCRIPTION
**Pull request by ErinUntermeyer**
# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
- I updated apiCalls to use promise and .then chaining instead of async and await.
- I updated getLocationData to return the data object to used when a user searched a location.
- I put some initial structure in Location so when we fetch data we can display it on the page
- I wrote a method in Search to get the input, use that to get the coordinates, and pass those into getLocationData to get data.
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Other, Please explain:
- The airvisual API is not allowing us to fetch data right now, there is a CORS error. I'm going to pivot and work with the other weather API we have to see if I can get some data displayed when a user searches for a location.

## Has This Been Tested?
- [x] No
- [ ] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
- I haven't written any tests yet because of the snag with the data. As soon as we can get some data displaying I will write some tests that the components are doing what they should be.

# Message for reviewer:
- Although there's not a lot of successful working methods here I think it's good to get back on the same page as far as what codebase we are all working on. I know we've all made quite a few changes! I will continue to work on getting the Location component to render some data when a user searches for a city once we update master.

# Checklist:
- [ ] My code has no unused/commented out code
- [ ] My code has no console logs
- [x] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/App/App.tsx [hunk 1]
```diff
@@ -1,32 +1,30 @@
...
 import Header from '../Header/Header'
 import View from '../View/View'
-import { getCoordinates } from '../helpers/apiCalls'
 
 const App: React.FC = () => {
-	// const [ topCities, setTopCities ] = useState([])
+	const [ topCities, setTopCities ] = useState([])
 
-	useEffect(() => {getTop5Cities()}, [])
+	// useEffect(() => {getTop5Cities()}, [])
```
Comment (ErinUntermeyer) [inline src/App/App.tsx:9]: Lines 9-18 can get deleted if y'all are cool with it?

File: src/App/App.tsx [hunk 2]
```diff
@@ -1,32 +1,30 @@
...
 
-	const getTop5Cities = async ():Promise<any> => {
-		const topCities = ['Denver']
-		const topCitiesData = topCities.map(city => {
-			return getCoordinates(city)
-		})
-		return topCitiesData
-	}
+	// const getTop5Cities = async ():Promise<any> => {
+	// 	const topCities = ['Denver', 'San Francisco']
```
Comment (leighlars) [inline src/App/App.tsx:12]: Good work jumping from ES5 to async await. I know that was a headache.

File: src/Location/Location.tsx [hunk 3]
```diff
@@ -1,6 +1,15 @@
-import React from "react";
-import * as ReactDOM from "react-dom";
-import "./Location.scss";
+import React from 'react'
+import './Location.scss'
 
+const Location: React.FC = () => {
```
Comment (leighlars) [inline src/Location/Location.tsx:4]: Nice, we can work with the styling on this

Comment (ErinUntermeyer) [reply src/Location/Location.tsx:4]: YES PLEASE! I would love that.

File: src/Search/Search.tsx [hunk 4]
```diff
@@ -16,18 +17,24 @@ const Search: React.FC = () => {
...
   }, [query])
 
-  useEffect(() => {
-    //async function for fetch call from debounced term and from api
-  })
+	useEffect(() => {
+		getSearchInputData()
+	}, [debouncedQuery])
+
+	const getSearchInputData = async () => {
```
Comment (leighlars) [inline src/Search/Search.tsx:24]: right on, once the API issue is resolved, we can really make waves.

File: src/App/App.tsx (hunk 1 above)
Comment (JoshSevy) [reply src/App/App.tsx:9]: set them free. Code is cheap.

File: src/App/App.tsx (hunk 2 above)
Comment (JoshSevy) [reply src/App/App.tsx:12]: Great work!

**Review by leighlars — APPROVED** [review]
Great work!

**Pull request merged**